tests/test_laser_profiles.py

```diff
@@ -1,17 +1,13 @@
-# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-A
 
 import pytest
 
 from lasy.laser import Laser
-from lasy.profiles.combined_profile import CombinedLongitudinalTransverseProfile
-from lasy.profiles.gaussian_profile import GaussianProfile
-from lasy.profiles.longitudinal.gaussian_profile import GaussianLongitudinalProfile
-from lasy.profiles.transverse.laguerre_gaussian_profile import (
-    LaguerreGaussianTransverseProfile,
-)
-from lasy.profiles.transverse.super_gaussian_profile import (
-    SuperGaussianTransverseProfile,
-)
+from lasy.profiles import CombinedLongitudinalTransverseProfile
+from lasy.profiles import GaussianProfile
+from lasy.profiles.longitudinal import GaussianLongitudinalProfile
+from lasy.profiles.transverse import LaguerreGaussianTransverseProfile
+from lasy.profiles.transverse import SuperGaussianTransverseProfile
```

Comment:
I guess the last two imports can be done in one call

Reply:
thanks, should be addressed